dlf/plugins/search/class.tx_dlf_search.php

```diff
@@ -762,7 +777,7 @@ public function makeFacetsMenuArray($content, $conf) {
 		$search['params']['facet.limit'] = $this->conf['limitFacets'];
 
 		// Perform search.
-		$results = $solr->service->search($search['query'], 0, $this->conf['limit'], $search['params']);
+		$results = $solr->service->search($search['query'], 0, 0, $search['params']);
 
```

Comment:
You are removing the result limitation for Solr which could potentially lead to big result sets. Is this intentional?

Reply:
I didn't remove the limit. The limit is set to 0 because we don't need any result beside the facets.